File: Modul-2/Soal-4/PRAK204-2410817220030-RACHELWINAYUDA.c
#include <stdio.h>
#include <math.h>

int main () {
    float jarijari, tinggi;

    scanf("%f", &jarijari);
    scanf("%f", &tinggi);

    float volume = 22.0/7 * pow(jarijari, 2) * tinggi;
    float luas = 2 * 22.0/7 * jarijari * (jarijari + tinggi);
    float keliling = 2 * 22.0/7 *jarijari;

    printf("Volume = %.2f\n", (volume));
    printf("Luas = %.2f\n", (luas));
    printf("Keliling = %.2f", (keliling));

    float jarijari1, tinggi1;

    scanf("%f", &jarijari1);
    scanf("%f", &tinggi1);

    float volume1 = 22.0/7 * pow(jarijari1, 2) * tinggi1;
    float luas1 = 2 * 22.0/7 * jarijari1 * (jarijari1 + tinggi1);
    float keliling1 = 2 * 22.0/7 * jarijari1;

    printf("Volume = %.2f\n", volume1);
    printf("Luas = %.2f\n", luas1);
    printf("Keliling = %.2f", keliling1);
    return 0;
}

File: Modul-2/Soal-4/PRAK204-2410817220030-RACHELWINAYUDA.py
jarijari = float(input())
tinggi = float(input())

volume = 22 / 7 * jarijari**2 * tinggi
luas = 2 * 22 / 7 * jarijari * (jarijari + tinggi)
keliling = 2 * 22 / 7 * jarijari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

jarijari1, tinggi1 = map(float, input().split())

volume1 = 22 / 7 * jarijari1**2 * tinggi1
luas1 = 2 * 22 / 7 * jarijari1 * (jarijari1 + tinggi1)
keliling1 = 2 * 22 / 7 * jarijari1

print(f"Volume = {volume1:.2f}")
print(f"Luas = {luas1:.2f}")
print(f"Keliling = {keliling1:.2f}")